docs/guides/how-to-reset-state.md

```diff
@@ -47,33 +44,40 @@ const useSlice = create<State & Actions>()((set, get) => ({
 Resetting multiple stores at once
 
 ```ts
-import { create as _create, StateCreator } from 'zustand'
+import { create as _create, type StateCreator } from 'zustand'
```

Comment:
This [verbatimModuleSyntax](https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax) is not in Zustand's [tsconfig.json](https://github.com/pmndrs/zustand/blob/main/tsconfig.json).

Because in the documentation we follow Zustand's code style configuration, maybe splitting this import will do?

In Zustand split imports (import + import type) [are used](https://github.com/search?q=repo%3Apmndrs%2Fzustand+%2Fimport+%28%5BA-z%5D%2B%2C%29%3F+%5C%7B+.*type+%7Cimport+type%2F&type=code), e.g., in `src/middleware/immer.ts` or in `src/context.ts`.

```suggestion
import { create as _create } from 'zustand'
import type { StateCreator } from 'zustand'
```